### ISSUE TITLE
Flushing volume only shows 2 digits

### ISSUE THREAD

**Issue by longrackslabs** [bambulab/BambuStudio#3640]
### Bambu Studio Version

1.8.4.51

### Where is the application from?

Bambu Lab github releases

### OS version

Linux Mint 21.3 Cinnamon

### Additional system information

![image](https://github.com/bambulab/BambuStudio/assets/1175466/46441370-5550-4d7e-98b5-bd2cdaa8ccd0)

### Printer

Bambu Lab X1C

### How to reproduce

1. Open Flushing Volumes window
2. Enter 3 digit flushing volume (I need 400 for example)
3. Flushing volume shows '00' - looks like text box is too small

### Actual results

3. Flushing volume shows '00' - looks like text boxes are too small
![Screenshot from 2024-02-23 09-29-38](https://github.com/bambulab/BambuStudio/assets/1175466/e35b0887-0040-4a2b-995f-d77c882fcd2a)

### Expected results

3. Flushing volume should show value entered, in my case '400'

### Project file & Debug log uploads

[Flushing_Volumes.zip](https://github.com/bambulab/BambuStudio/files/14388352/Flushing_Volumes.zip)

### Checklist of files to include

- [ ] Log file
- [X] Project file

**Comment by nadnerb33** [comment]
I have the same issue with PopOS (Ubuntu) 20.04 and every version of Bambu Studio I have tried (appimage, flatpak etc) including the latest flatpak version 1.9.1.66

Changing the OS text size has some influence, but the boxes are still too small.

Default text size
![image](https://github.com/bambulab/BambuStudio/assets/7316130/3659f1ec-41d0-4f07-8124-55e64395a62c)

Large text size:
![image](https://github.com/bambulab/BambuStudio/assets/7316130/aab6a85e-6f24-48c4-b6f5-f55d9978836a)

**Comment by Ralf-3D** [comment]
I have the same problem.

BambuStudio AppImage version 1.9.1.66.

LinuxMint 21.3 xfce see folowing system-info:
System:
  Kernel: 5.15.0-112-generic x86_64 bits: 64 compiler: gcc v: 11.4.0 Desktop: Xfce 4.18.1
    tk: Gtk 3.24.33 info: xfce4-panel wm: xfwm 4.18.0 vt: 7 dm: LightDM 1.30.0
    Distro: Linux Mint 21.3 Virginia base: Ubuntu 22.04 jammy

This bug also occures in OrcaSlicer OrcaSlicer 2.0.0 AppImage

![Flushing-Volumes_Werte-Felder-zu-kurz_v1](https://github.com/bambulab/BambuStudio/assets/83096698/3e0577ae-d963-4d4c-9f4b-3728e5bef471)

**Comment by MklBambu** [comment]
Sorry, it is a issue with wxWidgets.
we will do our best to solve it

**Comment by KeluDev-Code** [comment]
Hi, I have the same problem.
Is there any update on this issue?
Maybe a minimum width could be set

**Comment by andruhon** [comment]
Same issue 
Arch with KDE
![Image](https://github.com/user-attachments/assets/bcdeaa53-9f03-43d7-8599-024fcf6faafe)
Tried to force the popup to fullscreen, but it didn't help

![Image](https://github.com/user-attachments/assets/6d16d0b4-22d5-42a8-8054-59171f87f88d)

**Comment by andruhon** [comment]
Looks better in version 2.0.0.95

![Image](https://github.com/user-attachments/assets/a2af9a45-dd8c-424d-9184-dbad63ead596)

**Comment by XunZhangBambu** [comment]
@andruhon @KeluDev-Code @longrackslabs @nadnerb33 @Ralf-3D 
In the latest version, we are using a new approach to display this interface, and this should resolve your issue.

**Comment by longrackslabs** [comment]
Yes, I installed 2.0 and the new UI for this looks much better now.

I'm happy to close this issue.
